plugin/src/main/kotlin/me/tylerbwong/gradle/metalava/task/MetalavaCheckCompatibilityTask.kt

```diff
@@ -72,24 +72,27 @@ internal abstract class MetalavaCheckCompatibilityTask @Inject constructor(
             val tempFilename = project.layout.buildDirectory
                 .file(METALAVA_CURRENT_PATH).get().asFile.absolutePath
             val taskName = getFullTaskName(TASK_NAME, variantName)
-            val metalavaClasspath = project.getMetalavaClasspath(objectFactory, extension)
+            val metalavaClasspath = project.getMetalavaClasspath(
+                objectFactory,
+                jarPath = extension.metalavaJarPath.get().ifEmpty { null },
```

Comment:
Personally, I'd just pass in the propery from the extension using get() and do the `.ifEmpty { null }` in the implementation to avoid duplicating the ifEmpty part